test/quoteSpec.js

```diff
@@ -362,6 +364,44 @@ describe('Quotes', function () {
         .end()
     })
 
+    it('caches source and destination ledger precision', function * () {
+      nock.cleanAll()
+      nock('http://eur-ledger.example/EUR')
+        .get('').reply(200, {precision: 10, scale: 4})
+        .get('').reply(500, 'Invalid request')
```

Comment:
I don't think that does what we want here. I don't see any way in nock to create a negative expectation.